<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggerWindow</class>
 <widget class="QWidget" name="LoggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../data/resources/mainwindow.qrc">
    <normaloff>:/images/leap-color-big.png</normaloff>:/images/leap-color-big.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QTextBrowser" name="txtLogHistory"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnDebug">
       <property name="text">
        <string>Debug</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/script-error.png</normaloff>:/images/oxygen-icons/script-error.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInfo">
       <property name="text">
        <string>Info</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/dialog-information.png</normaloff>:/images/oxygen-icons/dialog-information.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWarning">
       <property name="text">
        <string>Warning</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/dialog-warning.png</normaloff>:/images/oxygen-icons/dialog-warning.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnError">
       <property name="text">
        <string>Error</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/dialog-error.png</normaloff>:/images/oxygen-icons/dialog-error.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCritical">
       <property name="text">
        <string>Critical</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/edit-bomb.png</normaloff>:/images/oxygen-icons/edit-bomb.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save to file</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/document-save-as.png</normaloff>:/images/oxygen-icons/document-save-as.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnDebug</tabstop>
  <tabstop>btnInfo</tabstop>
  <tabstop>btnWarning</tabstop>
  <tabstop>btnError</tabstop>
  <tabstop>btnCritical</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>txtLogHistory</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../data/resources/loggerwindow.qrc"/>
  <include location="../../../../data/resources/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
